ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -17,134 +17,143 @@
     <p>{{t 'resources.session.description'}}</p>
   </page.header>
 
-  <page.body>
-    <Rose::Toolbar as |toolbar|>
-      <toolbar.primary>
-        <Rose::Form as |form|>
-          {{#let
-            (resource-filter 'scopes.scope.sessions' 'status')
-            as |filter|
-          }}
-            <Rose::Dropdown
-              @text={{t (concat 'form.' filter.name '.label')}}
-              @count={{filter.selectedValue.length}}
-              @ignoreClickInside={{true}}
-              as |dropdown|
+  <page.body class='search-filtering'>
+    {{#if @model.allSessions}}
+      <div class='search-filtering-toolbar'>
+        <Hds::SegmentedGroup data-test-sessions-bar as |S|>
+          <S.TextInput
+            @value={{this.search}}
+            @type='search'
+            placeholder={{t 'actions.search'}}
+            aria-label={{t 'actions.search'}}
+            {{on 'input' this.handleSearchInput}}
+          />
+          <S.Generic>
+            <Dropdown
+              @name={{t 'resources.user.title'}}
```

Comment:
none blocking comment: should we use translated string for `@name`? have in mind i18n strings will change the value and are Uppercase, depending of the usage you want for this attribute I will consider hardcoding the value.

Reply:
That is a good point. We are using translations for this in a few places. I will add this as a tech time topic to discuss.